service-workers/service-worker/navigationpreload.https.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>ServiceWorker: navigator.serviceWorker.navigationPreload</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="resources/test-helpers.sub.js"></script>
+<script>
+promise_test(async t => {
+    const SCRIPT = 'resources/empty-worker.js';
+    const SCOPE = 'resources/navigationpreload';
+    let registration =
```

Comment:
const

Reply:
This test should be move to directory `navigation-preload` and this problem should not exist

Reply:
Oh I see. I think you need to remove this file from the pull request maybe?